### PR TITLE
feat: add event to load-more-pager for when new items are loaded

### PR DESCRIPTION
**Pull request by dlockhart**
I found myself needing to know when the items had actually loaded when writing a vdiff test.

### CODE REVIEW

File: components/paging/pager-load-more.js [hunk 1]
```diff
@@ -129,20 +130,33 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
...
+				if (firstFocusable) {
+					itemToFocus = firstFocusable;
+				} else if (item.focus === nativeFocus) {
+					item.tabIndex = -1;
+					itemToFocus = item;
+				}
 			}
 		}
+
+		if (itemToFocus) {
```
Comment (dlockhart) [inline components/paging/pager-load-more.js:149]: Just a slight refactor here to centralize the wait for an animation frame and the focus calls.

File: components/paging/pager-load-more.js [hunk 2]
```diff
@@ -129,20 +130,33 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
...
+			await new Promise(requestAnimationFrame);
+			itemToFocus.focus();
+		}
+
+		await new Promise(setTimeout);
```
Comment (dlockhart) [inline components/paging/pager-load-more.js:154]: This timeout is the ensure that focus has moved... should it be an animation frame instead?

Comment (dbatiste) [reply components/paging/pager-load-more.js:154]: I think so yeah.

Btw, I noticed a line further above that could probably be simplified.

```javascript
await new Promise(resolve => requestAnimationFrame(resolve));
```
Pretty sure I wrote that... not sure why I didn't write it the way you did on line 150. I can't imagine the timing being different.

Comment (dlockhart) [reply components/paging/pager-load-more.js]: Cool -- fixed.

File: components/paging/pager-load-more.js [hunk 3]
```diff
@@ -129,20 +130,33 @@ class LoadMore extends PageableSubscriberMixin(FocusMixin(LocalizeCoreElement(Rt
...
+		this.dispatchEvent(new CustomEvent('d2l-pager-load-more-loaded', {
+			bubbles: false,
```
Comment (dbatiste) [inline components/paging/pager-load-more.js:156]: Nit: these [are false by default](https://developer.mozilla.org/en-US/docs/Web/API/Event/Event#options), so if inclined, this statement could become:
```javascript
this.dispatchEvent(new CustomEvent('d2l-pager-load-more-loaded'));
```

**Review by dbatiste — APPROVED** [review]

**Pull request merged**